### PR TITLE
Add Exercise04

### PR DESCRIPTION
**Pull request by huynhngocminh2511**

### CODE REVIEW

File: Exercise04/Program.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+{
+    public class Program
+    {
+        public static string filePath = "../../../JsonFile/employees.json";
+
+        static void Main(string[] args)
+        {
+            Console.Write("FilePath of JsonFile: ");
+            var str = Console.ReadLine();
+            if (!str.Equals(""))
```
Comment (voquanghoa) [inline Exercise04/Program.cs]: `!string.IsNullOrEmpty`

File: Exercise04/Program.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                filePath = str;
+            }
+            var jsonString = File.ReadAllText(filePath);
+
+            var obj = JsonConvert.DeserializeObject<Employess>(jsonString);
+            var xmlSerializer = new XmlSerializer(obj.GetType());
+
+            var xmlString = "";
+
+            using (var strWriter = new StringWriter())
```
Comment (voquanghoa) [inline Exercise04/Program.cs]: Extract method

File: Exercise04/Program.cs [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+            {
+                var writer = XmlWriter.Create(strWriter);
+                xmlSerializer.Serialize(writer, obj);
+                xmlString = strWriter.ToString();
+            }
+
+            Console.WriteLine(xmlString);
+
+            var xmldoc = new XmlDocument();
```
Comment (voquanghoa) [inline Exercise04/Program.cs]: Extract method

File: Exercise04/Program.cs (hunk 1 above)
Comment (huynhngocminh2511) [reply Exercise04/Program.cs]: Done

File: Exercise04/Program.cs (hunk 2 above)
Comment (huynhngocminh2511) [reply Exercise04/Program.cs]: Done

File: Exercise04/Program.cs (hunk 3 above)
Comment (huynhngocminh2511) [reply Exercise04/Program.cs]: Done

**Pull request merged**